routes/dashboard.js

```diff
@@ -21,12 +21,28 @@ router.post('/userinfo',verify.isAuthenticated, function(req, res, next) {
           console.log(req);
           //var userinfo = {id:123,AadharId:123456}//req.body//applicant_mapping(req.body);
           //console.log(req.body);
+    console.log('inside userfingo')
+    console.log(req.body);
+          var userinfo = {
+              "FirstName" : req.body.first_name,
+              "LastName" : req.body.last_name,
+              "Discipline" : req.body.cse
+          }
+
           //Transaction started.
+    console.log(userinfo)
           models.sequelize.transaction(t=>{
             //Transaction Is used for only one session per user.
-            return models.applicant.create(userinfo).then(userinfo=>{
```

Comment:
What if an applicant if first time sending data to /userinfo route.  How can query find data which is not present in database.